frontend/src/locales/da/input_fields.json

```diff
@@ -17,6 +17,8 @@
   "name": "Navn",
   "new_name": "Nyt navn",
   "org_unit_type": "@:shared.org_unit_type",
+  "user_key": "Brugervendt nøgle",
```

Comment:
I think we should check with Alex or perhaps a customer how they refer to this; “brugervendt nøgle” seems overly technical to me. Perhaps “Kort navn” or (ew) “Enhedsnummer”?

Reply:
Agree! - han har fået skærmbilledet med placeholder og hele molevitten lige op i raketten

Reply:
Han var glad for skærmbilledet - og Brugervendt nøgle er jo det vi kalder den? Eller er det Enhedsnummer nu? Hvad så med medarbejdere - det er jo samme koncept?